.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 * * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        debug: true
+        days-before-stale: 30
+        days-before-close: 5
+        stale-issue-message: 'Stale issue message'
```

Comment:
Is this the message that will get posted?

Reply:
Yes, that was a placeholder that I didn't change. Will update.

Reply:
```suggestion
        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Please comment or update this issue or it will be closed in 5 days.'
```